src/Agent/NewRelic/Agent/Core/Configuration/ReportedConfiguration.cs

```diff
@@ -60,7 +60,7 @@ public class ReportedConfiguration
         public IDictionary<string, IEnumerable<string>> ErrorCollectorExpectedMessages { get; set; }
 
         [JsonProperty("error_collector.expected_status_codes")]
-        public string ErrorCollectoryExpectedStatusCodes { get; set; }
+        public IEnumerable<string> ErrorCollectoryExpectedStatusCodes { get; set; }
```

Comment:
Can we fix the typo in the name?  Should be `ErrorCollectorExpectedStatusCodes`

Reply:
👍